morerss/zhihu_stream.py

```diff
@@ -252,20 +262,20 @@ class ZhihuStream(base.BaseHandler):
     self.finish(rss)
 
 class ZhihuTopic(base.BaseHandler):
-  async def get(self, name):
+  async def get(self, id):
     """
-    :param name (str): Zhihu topic slug, as "19551894" in "https://www.zhihu.com/topic/19551894/hot"
+    :param id (str): Zhihu topic slug, as "19551894" in "https://www.zhihu.com/topic/19551894/hot"
```

Comment:
It's an id, not a slug. No doc is better than wrong doc....

Reply:
f669c1a fixes the term error.

Reply:
> 就是知乎新版个人页面的动态啦。不过只支持回答和文章两种类型。
https://rss.lilydjwg.me/

@lilydjwg I thought you ignore the question type by design. If I modify `morerss.zhihu_stream.post2rss` to process the questions, then users feeds like https://rss.lilydjwg.me/zhihu/jian-dan-xin-li will include questions too.

Should `morerss.zhihu_stream.post2rss` include questions? Or should I make this for topics only?